### ISSUE TITLE
Console window is always displayed when target Photino.Net app is run.

### ISSUE THREAD

**Issue by raddevus** [tryphotino/photino.NET#94]
I've [written this up at StackOverflow & created a new tag (photino.net)](https://stackoverflow.com/questions/71914882/how-do-i-build-deploy-a-photino-net-app-so-it-does-not-display-the-console-whe) there.

The problem is that every time I run my target Photino.NET app a Console window is also opened.
Is there a way to build the final Release version so that the console window is not displayed also?  (Only target app window is shown)

### What I've Tried

1. Read all documentation at [official Photino site](https://docs.tryphotino.io/Welcome-to-the-Photino-Docs).
2. Built for Win10 with the following parameters: `dotnet publish -r win10-x64 -p:PublishSingleFile=true --self-contained true`
3. Built for Linux (Ubuntu 21.04) with following parameters: `dotnet publish -r linux-x64 -p:PublishSingleFile=true --self-contained true`
4. Built other Photino samples (Photino.HelloPhotino.React) and they also display two windows (console & main target app)

I've ran both of the resulting EXEs on their appropriate OSes (win10, Ubuntu) and in both cases I get two windows:
1. Console window
2. Main Target app window

### The Main Issue
Is there are way to run the final release version so that the user **only sees the target app window** (no console window)?
**Note**: This same issue (two windows display) occurs on Win10 also.

Here's a snapshot of my app main window running (highlighted in green) with the Console window  (highlighted in red) behind it.
![Y1qJI](https://user-images.githubusercontent.com/16722666/163849195-034fd282-58ef-4744-b937-63bb180f5bdf.png)

**Comment by albertospelta** [comment]
You can change the project `OutputType` from `Exe` to `WinExe` and the console will not be launched.

```xml
<Project Sdk="Microsoft.NET.Sdk.Web">
  <PropertyGroup>
    <OutputType>Exe</OutputType>
  </PropertyGroup>
</Project>
```


![image](https://user-images.githubusercontent.com/1957788/163956783-8073d202-b41b-460d-8dc5-0e7eb9e07c7b.png)

**Comment by raddevus** [comment]
That did indeed work on Windows.
The default project .csproj file (from the Photino template) has :

```
<PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net5.0</TargetFramework>
    <Platforms>AnyCPU;x64</Platforms>
  </PropertyGroup>
```

I altered it and changed Exe to WinExe and built it with the dotnet build command I showed above on Windows 10 & it did indeed launch the app and did **not** show the console window any longer.

It looks as if this does fix it on Linux also.  Seems odd since it is marked WinExe but that just seems to be the code for deploy as EXE.
Thanks for your help.
